.rubocop.yml

```diff
@@ -12,8 +12,10 @@ require:
 AllCops:
   Exclude:
     - 'bin/**/*'
-    - 'db/migrate/*'
+    - 'db/primary_migrate/*'
     - 'db/schema.rb'
+    - 'db/worker_jobs_migrate/*'
+    - 'db/worker_jobs_schema.rb'
```

Comment:
prod *should* be good to go now, but we should watch migrations extra closely for this deploy